counterr/counters.py

```diff
@@ -89,15 +96,27 @@ def mapQ_stats_aligned_readsegment(reads, verbose):
     stds_out = []
     lens_out = []
     for name in reads.keys():
-        for read in reads[name]: # Iterate through the reads
+        for read in reads[name]:  # Iterate through the reads
             # Inside aligned region
             Q_in = read.query_alignment_qualities
-            means_in.append(np.mean(Q_in))
-            stds_in.append(np.std(Q_in))
+            # Q_in is None implies that there are no alignment qualities, so we
```

Comment:
This is the part which was actually giving an error. `read.query_alignment_qualities` was returning `None`, and this was gunking up computing a mean and standard deviation. If this is None, then I just append 0s.